src/Magnum/ImGuiIntegration/Integration.h

```diff
@@ -57,9 +57,25 @@ Example usage:
 #include "Magnum/ImGuiIntegration/visibility.h" /* defines IMGUI_API */
 
 #include <imgui.h>
+#include <Corrade/Containers/StringView.h>
 #include <Magnum/Types.h>
 #include <Magnum/Math/Vector.h>
 
+#ifndef DOXYGEN_GENERATING_OUTPUT
+#ifdef IMGUI_HAS_IMSTR
```

Comment:
```suggestion
/* Currently present only in the features/string_view branch of ImGui */
#ifdef IMGUI_HAS_IMSTR
```

... or something like that, to make it clear why the ifdef is there.